### ISSUE TITLE
Dupe detection doesn't seem to work

### ISSUE THREAD

**Issue by rnewman** [mozilla-services/readinglist#82]
I have a test that uploads records to a throwaway account.

```
    // Upload a record.
    final ReadingListRecord record = new ReadingListRecord("http://reddit.com", "Reddit", "Test Device");
    Assert.assertEquals(record.url, "http://reddit.com");
    Assert.assertEquals(record.title, "Reddit");
    Assert.assertEquals(record.addedBy, "Test Device");
```

The upload succeeds with a 201 each time. An un-limited fetch returns every record I've uploaded — up to 20 now — each of which was created with the same inputs.

Shouldn't all but the first have been a conflict, given that the URL wasn't unique?

**Comment by leplatrem** [comment]
Yes of course it should ! Once issue #20 is implemented and deployed :)

**Comment by Natim** [comment]
Fixed in #88
